style.css

```diff
@@ -3,13 +3,6 @@ body {
     box-sizing: border-box;
 }
 
-.main-content input.search-input {
```

Comment:
@ChrysaLynn 
I noticed here (and I _think_ that this was my fault anyway) that we had conflicting selectors. There was already a set of items below that were for `input.search-input` and because _this_ one was **_more_** specific...it was overriding any changes made to the one below.